client.go

```diff
@@ -129,7 +138,8 @@ func (c *Client) send(line string) {
 			time.Sleep(time.Millisecond * 2)
 			continue
 		}
-		fmt.Fprint(*c.connection, line+"\r\n")
+		//fmt.Fprint(*c.connection, line+"\r\n")
```

Comment:
Comments should be removed. I prefer the direct way of writing, instead of using fmt. That's a good change.